src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -292,7 +292,7 @@ now be added using the *Create Node Source* panel in the Resource Manager or the
     $ PROACTIVE_HOME/bin/proactive-client --createns openstack -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.OpenstackInfrastructure username password endpoint rmHostname connectorIaasURL image flavor publicKeyName numberOfInstances numberOfNodesPerInstance downloadCommand additionalProperties
 
 
-WARNING: When the Scheduler is stopped (without removing the created infrastructure), the Openstack instances will not be terminated. And when the Scheduler is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the Openstack server side, they will be re-created and re-configured.
+WARNING: When the ProActive server is stopped (without removing the created infrastructure), the Openstack instances will not be terminated. And when the ProActive server is restarted, the infrastrucutre will be re-configured as per previous settings. If the instances were delete at the Openstack server side, they will be re-created and re-configured.
```

Comment:
- When ProActive server is stopped (without removing the created infrastructure), Openstack instances 
- when ProActive server is restarted, 
- will be re-configured as per **the** previous settings. 
-If the instances were **deleted** at the Openstack **Cloud** side,